.github/maintainers_guide.md

```diff
@@ -1,199 +1,162 @@
-# Maintainers Guide
+# Maintainer's Guide
 
-> ⚠️ This maintainers guide is out of date. Progress on updating it is being [tracked](https://github.com/slackapi/node-slack-sdk/issues/748).
+This document describes tools, tasks and workflow that one needs to be familiar with in order to effectively maintain this project. If you use this package within your own software but don't plan on modifying it, this guide is **not** for you. Please refer to the [Contributor's Guide](https://github.com/slackapi/node-slack-sdk/blob/main/.github/contributing.md).
 
-This document describes tools, tasks and workflow that one needs to be familiar with in order to effectively maintain
-this project. If you use this package within your own software but don't plan on modifying it, this guide is
-**not** for you.
+## 🛠 Tools
+Maintaining this project requires installing [Node.js](https://nodejs.org). All of the remaining tools are downloaded as `devDependencies`, which means you'll have them available once you run `npm install` in a working copy of this repository.
 
-## Tools
+## ✅ Tasks
 
-Maintaining this project requires installing [Node.js](https://nodejs.org) in your development environment. All of the
-remaining tools are downloaded as `devDependencies`, which means you'll have them available once you run `npm install`
-in a working copy of this repository.
+### ⚗️ Testing and Linting
+To run tests for a given package, navigate to the package directory and run `npm test`. To run linting, run `npm run lint`.
 
-In particular, [`lerna`](https://lerna.js.org/) is a tool you should become familiar with. This project is home to
-several packages, and lerna makes it easier to manage operations across the set of packages. The very next thing you
-should do is run `npx lerna bootstrap` in the root of this project to install dependencies for each of the packages,
-and to allow lerna to link them to each other.
+This project has tests for individual packages as `*.spec.js` files and inside of each's respective `test` directory. It also has integration tests in the `integration-tests` directory under the root `support` directory. 
 
-## Tasks
+Upon opening a PR, tests are executed by GitHub Actions, our continuous integration system. GitHub Actions runs several, more granular builds in order to report on success and failure in a more targeted way. 
 
-### Testing
-
-This project has tests for individual packages as `*.spec.js` files and inside of each's respective `test` directory. It also has
-integration tests in the `integration-tests` directory under the root `support` directory. You can run the entire test
-suite using the npm script `npm test` at the top level. This script will use Lerna to invoke tests in each package and
```

Comment:
I took a look because I wanted to make sure I didn't overlook anything, but `lerna` is only referenced at the root level `package.json`: none of the individual packages rely on it. I would have removed it entirely, but Steve hypothesized that we might cause CI errors, so I've omitted that from this PR.

Are you running these commands from the root level?